ffi/ffi.cpp

```diff
@@ -525,12 +669,20 @@ extern "C" lean_obj_arg termManager_getStringSort(lean_obj_arg tm)
 
 extern "C" lean_obj_arg termManager_mkArraySort(lean_obj_arg tm, lean_obj_arg idx, lean_obj_arg elm)
 {
-  return sort_box(new Sort(mut_tm_unbox(tm)->mkArraySort(*sort_unbox(idx), *sort_unbox(elm))));
+  CVC5_TRY_CATCH_EXCEPT(
+    return except_ok(lean_box(0),
+      sort_box(new Sort(mut_tm_unbox(tm)->mkArraySort(*sort_unbox(idx), *sort_unbox(elm))))
+    );
+  )
 }
 
 extern "C" lean_obj_arg termManager_mkBitVectorSort(lean_obj_arg tm, uint32_t size)
 {
-  return sort_box(new Sort(mut_tm_unbox(tm)->mkBitVectorSort(size)));
+  CVC5_TRY_CATCH_EXCEPT(

```

Comment:
This try/catch block may not be needed (same for some other functions)

Reply:
As discussed above, `mkBitVectorSort` fails (at least) on `0`.